src/jti_content.js

```diff
@@ -20,6 +20,10 @@ if (navigator.userAgent.indexOf('Firefox') !== -1 || navigator.userAgent.indexOf
 // Handle <TI> tag selection.
 $(document).on('click', '#description', function () {
     var text = $(this).val(),
+        ctrlDown = false,

```

Comment:
Sorry. Meant to write "I don't think this file should be modified" you should be able to reset it.